references.bib

```diff
@@ -74,42 +59,28 @@ @Article{dimension-reduction-guidelines
   url = {https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1006907&type=printable},
 }
 
-@Manual{ggplot2,
+@Website{pca-visually-explained,
+  title = {Principal Component Analysis Explained Visually},
+  author = {Victor Powell and Lewis Lehe}, 
+  url = {https://setosa.io/ev/principal-component-analysis/},
+}
+
+@Manual{Prabhakaran2016,
   title = {The Complete ggplot2 Tutorial},
   author = {Selva Prabhakaran},
   year = {2016-2017},
   url = {http://r-statistics.co/Complete-Ggplot2-Tutorial-Part1-With-R-Code.html},
 }
 
-@Manual{make-complex-heatmaps, 
-  title = {ComplexHeatmap Complete Reference}, 
-  author = {Zuguang Gu},
-  year = {2020}, 
-  url = {https://jokergoo.github.io/ComplexHeatmap-reference/book/}, 
-}
-
-@Article{make-heatmaps,
-  title = {Make heatmaps in R with pheatmap},
-  author = {Kamil Slowikowski},
-  year = {2017},
-  url = {https://slowkow.com/notes/pheatmap-tutorial/},
-}
-
-@Website{pca-explained,
-  title = {Principal Component Analysis Explained Visually},
-  author = {Victor Powell and Lewis Lehe}, 
-  url = {https://setosa.io/ev/principal-component-analysis/},
-}
-
-@Manual{pheatmap, 
+@Manual{Kolde2019, 
   title = {pheatmap: Pretty Heatmaps}, 
   author = {Raivo Kolde},
   note = {R CRAN version 1.0.12},
   year = {2019}, 
   url = {https://CRAN.R-project.org/package=pheatmap}, 
 }
 
-@Manual{R-base,
+@Manual{Rbase,
```

Comment:
So we are essentially treating this as one word? Which then would not need to obey the guideline in `CONTRIBUTING.md` that states `If there is no author name or no year associated with the reference, then turn to key words, all lower case with `-` in between.`.

Is that correct? Just wanted to check this concept.

Reply:
Yeah, you are right. I think the dash should be maintained.